### DISCUSSION TITLE
[#991887] Mobs will attack you out of Crosshair Mode

### DISCUSSION THREAD

**Discussion by ihategravel2** [Mojang/minecraft-editor#13] [Bugs]
If you set your gamemode to survival and exit Crosshair mode mobs will still attack you.

https://twitter.com/ihategravel2/status/1633537715617554440

**Comment by minecraftchloe** [comment]
Thank you! Updating title to reflect internal tracking IDs for ease.

**Comment by rpoerschke** [comment]
Fixed in 0.3.3.  Thank you!
